product_docs/docs/biganimal/release/using_cluster/managing_cloud_account/csp_tagging.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: "Tagging AWS resources"
+deepToC: true
+---
+
+Tags are key-value pairs you can apply to cloud provider resources created on your own cloud account (Your Cloud Account). Resource tagging allows you to have fine-grained access control and granular cost data at the bucket level. When you add a tag, the following resources are labeled: 
+
+- EC2 instances
+- S3 buckets
+- Load balancers
+- Volumes
+
+You can create and manage tags on a per-project basis in the EDB Postgres® AI Console. The Console then applies the tag to the resources of all regions configured in your account. 
+
+## Create Tags
+
+To create a tag:
+
+1.  Log in to the Console.
+    
+1.  Go to the project's page.
+
+1.  In the project's list, select your **Project**.
+
+1.  On your project's home page left navigation, select **Cloud Providers**.
+
+1.  On the cloud provider's page, select **Your Cloud Account**.
+
+1.  On your cloud account tab, under **AWS**, select **Manage Tags**.
+
+1.  Select **Add a Key Value pair**.
+
+1.  Provide **Key** and **Value** and select **Save**.
+
+1.  A **Confirm changes to AWS tags** message pops up.
+
+1.  Select **Confirm** to add the tag.
+
+1.  This **Key:Value** pair is added at your project level. It is populated on the EC2 instances, S3 buckets, load balancers and volumes managed in that project, for all regions and clusters.
+
+This new tag appears in the list under **AWS** on Cloud Service Providers. 
+
+This tag is also propagated on your AWS resources. For instance, you can view this tag in the **Tags** tab of your EC2 instance resources on the AWS console.
+
+### Considerations
+
+Consider following while adding a **Key:Value** pair:
+
+- Review the [tagging limitations of AWS](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/tagging-restrictions.html).
+
+- The **Key** cannot have `aws` or `managedby` in the prefix, as they are reserved by AWS and EDB Cloud Service. 
+
+- In AWS, a resource can have a maximum number of 50 tags. The EDB Cloud Service applies some tags by default, so we recommend not adding more than 40 of your own tags. Take into account that the number of tags per resource is defined by the tags created in the EDB Console, in your cloud service provider's dashboard, with terraform, and other synced systems.
```

Comment:
Based on the discussion in #dbaas-incubator, it seems like we want to lower the number of customer-added tags to 20? 

@amine-khemissi, @dhope-nagesh or @navnathedb  to confirm, please 🙏 

```suggestion
- In AWS, a resource can have a maximum number of 50 tags. The EDB Cloud Service and AWS apply some tags by default, so we recommend not adding more than 20 of your own tags. Take into account that the number of tags per resource is defined by the tags created in the EDB Console, in your cloud service provider's dashboard, with terraform, and other synced systems.
```

Reply:
yeah, I think we can start with just 20 cc @marie-linet  @navnathedb

Reply:
yes, 20 is a decent number to start with.